server/src/projects/projects.service.ts

```diff
@@ -10,6 +10,14 @@ const APPLICANT_ACTIVE_STATUS_CODE = 1;
 const PROJECT_ACTIVE_STATE_CODE = 0;
 const PROJECT_VISIBILITY_APPLICANT_ONLY = 717170002;
 const PROJECT_VISIBILITY_GENERAL_PUBLIC = 717170003;
+const PACKAGE_VISIBILITY_APPLICANT_ONLY = 717170002;
```

Comment:
Since this list is getting long, one nice thing you can do to organize the code is placing these under an object. like:
```js
 PACKAGE_VISIBILITY = {
    APPLICANT_ONLY: 71717003,
    GENERAL_PUBLIC: 71717004,
    ...
}
```